prettyprinter/src/Data/Text/Prettyprint/Doc/Render/Util/StackMachine.hs

```diff
@@ -59,13 +59,13 @@ import Data.Monoid
 --
 -- >>> let doc = "hello" <+> annotate () "world" <> "!"
 -- >>> let sdoc = layoutPretty defaultLayoutOptions doc
--- >>> T.putStrLn (renderSimplyDecorated id (\() -> ">>>") (\() -> "<<<") sdoc)
+-- >>> T.putStrLn (renderSimplyDecorated (const id) (\() -> ">>>") (\() -> "<<<") sdoc)
 -- hello >>>world<<<!
 --
 -- The monoid will be concatenated in a /right associative/ fashion.
 renderSimplyDecorated
     :: Monoid out
-    => (Text -> out) -- ^ Render plain 'Text'
+    => ([ann] -> Text -> out) -- ^ Render plain 'Text'
```

Comment:
The haddocks should say what the `[ann]` argument is.

Reply:
Fair point. Will add that if this gets merged.